cloudbees-ci/configuration-as-code-examples/cloudbees-plugin-usage-plugin/configuration-as-code.yml

```diff
@@ -0,0 +1,3 @@
+tool:
+  cloudbeesPluginUsageAnalyzer:
+    enabled: true
```

Comment:
This is the default configuration, so maybe we should change it to `enabled: false` instead?

```suggestion
    enabled: false
```

Reply:
Yeah I think it makes more sense to display a fragment you would actually use.